DigitalOffice/Tests/TimeManagementServiceTests/Mappers/WorkTimeMapperTests.cs

```diff
@@ -0,0 +1,81 @@
+using NUnit.Framework;
+using System;
+using LT.DigitalOffice.TimeManagementService.Database.Entities;
+using LT.DigitalOffice.TimeManagementService.Mappers;
+using LT.DigitalOffice.TimeManagementService.Mappers.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementServiceUnitTests.UnitTestLibrary;
+
+namespace LT.DigitalOffice.TimeManagementServiceUnitTests.Mappers
+{
+    public class WorkTimeMapperTests
+    {
+        private IMapper<CreateWorkTimeRequest, DbWorkTime> mapper;
+
+        private CreateWorkTimeRequest request;
+
+        [SetUp]
+        public void SetUp()
```

Comment:
[OneTimeSetup]

Reply:
I use the fact that it fills up every time.